scripts/gen_openapi_spec.ts

```diff
@@ -26,14 +26,14 @@ const swaggerDefinition = {
     version: '1.0.0',
     description: 'A sample API',
```

Comment:
Can we change this before we forget it in the end? 😅 
```suggestion
    description: 'The JIMMI backend REST api',
```